builder

```diff
@@ -104,6 +104,8 @@ if [[ -z "$IMAGE_NAME" ]]; then
   IMAGE_NAME="${REPO}_${BRANCH}:${COMMIT}"
 fi
 
+IMAGE_NAME=${IMAGE_NAME//\//\-}
+
```

Comment:
I'm not sure that this would work here because of the push portion down below. It would change the `sfgg/leanin-web:master` image name to `sgff-leanin-web:master` which is not a valid account / image setup.

Is that correct @jgallen23 ?

Reply:
Addressed this!